driver/terraform.go

```diff
@@ -371,6 +358,19 @@ func (tf *Terraform) initTask() error {
 	// will reinit
 	tf.inited = false
 
+	// initialize workspace
+	taskName := tf.task.Name()
+	if err := tf.init(ctx); err != nil {
+		log.Printf("[ERR] (driver.terraform) error initializing workspace, "+
+			"skipping validate for '%s'", taskName)
```

Comment:
Minor nit: the "skipping..." wording comes from the fact that (for example) the terraform-apply didn't happen but will be retried on the next change. Since I think we're exiting if this init fails, the "skipping" language may not be necessary anymore.

Reply:
Makes sense, I'll remove it!